.circleci/config.yml

```diff
@@ -93,103 +45,98 @@ references:
           - v1-yarn-packages-cache
     - run:
         name: Install Yarn packages
-        command: |
-          if [ "${CIRCLE_NODE_INDEX}" == "0" ]; then
-            yarn --frozen-lockfile
-          fi
+        command: yarn --frozen-lockfile
     - save_cache:
         key: v1-yarn-packages-cache-{{ checksum "yarn.lock" }}
         paths:
           - node_modules
     - run:
         name: Run Rubocop
-        command: |
-          if [ "${CIRCLE_NODE_INDEX}" == "0" ]; then
-            bin/rails rubocop
-          fi
+        command:  bin/rails rubocop
     - run:
         name: Run erblint
-        command: |
-          if [ "${CIRCLE_NODE_INDEX}" == "0" ]; then
-            bin/rails erblint
-          fi
-    - run:
-        name: Run javascript tests
-        command: |
-          if [ "${CIRCLE_NODE_INDEX}" == "0" ]; then
-            bin/rails javascript_tests
-          fi
+        command: bin/rails erblint
     - run:
         name: Run ruby tests
-        command: |
-          if [ "${CIRCLE_NODE_INDEX}" == "0" ]; then
-            bin/rails spec
-          fi
+        command: bin/rails spec
     - run:
         name: Run integration tests
+        command: SAVE_PAGES=true bin/rails cucumber
+    - run:
+        name: Generate webhint reports
+        command: ./bin/generate_webhint_reports.sh
+    - run:
+        name: gzip webhint reports
+        command: tar -zcvf hint-report.tar.gz hint-report/
+    - store_artifacts:
+        path: ./hint-report.tar.gz
+        destination: webhint-reports
+    - run:
+        name: gzip html pages
+        command: tar -zcvf html-pages.tar.gz tmp/webhint_inputs/
+    - store_artifacts:
+        path: ./html-pages.tar.gz
+        destination: html-pages
+
+  build_and_push:
+    <<: *build_container_config
+    steps:
+    - checkout
+    - setup_remote_docker
+    - run:
+        name: Build docker images
+        command: docker build -t app .
+    - *setup_aws_login
+    - run:
+        name: Push docker image to ECR repo
         command: |
-          if [ "${CIRCLE_NODE_INDEX}" == "0" ]; then
-            SAVE_PAGES=true bin/rails cucumber
+          docker tag app "${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${REPO_NAME}:${CIRCLE_SHA1}"
+          docker push "${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${REPO_NAME}:${CIRCLE_SHA1}"
+
+          if [ "${CIRCLE_BRANCH}" == "master" ]; then
+            docker tag app "${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${REPO_NAME}:latest"
+            docker push "${ECR_ENDPOINT}/${GITHUB_TEAM_NAME_SLUG}/${REPO_NAME}:latest"
           fi
-    - *build_docker_image
-    - *push_to_ecr
-    - run: mkdir -p tmp/webhint_inputs
-    - persist_to_workspace:
-        root: tmp
-        paths:
-          - webhint_inputs
 
   deploy_uat:
-    <<: *deploy_container_config
+    <<: *cloud_platform_container_config
     steps:
     - checkout
-    - setup_remote_docker:
-        docker_layer_caching: true
```

Comment:
What's the reason for removing `docker_layer_caching`?

Reply:
Because it doesn't work at the moment. It's a CircleCI feature that needs to be activated on a project per-project basis and it costs £15.
https://mojdt.slack.com/archives/CCSD5F397/p1562319821001800

Reply:
Thanks 👍💰